lib/Client.js

```diff
@@ -588,6 +588,7 @@ class Client extends EventEmitter {
     * @arg {String} [content.messageReference.guildID] The guild ID of the referenced message
     * @arg {String} content.messageReference.messageID The message ID of the referenced message. This cannot reference a system message
     * @arg {String} [content.messageReferenceID] [DEPRECATED] The ID of the message should be replied to. Use `messageReference` instead
+    * @arg {Array<String>} [content.stickerIDs] An array of ids corresponding to stickers to send
```

Comment:
I didn't notice this earlier, but for consistency reasons, should `ids` be changed to `IDs` here? And the same for the JS docs in PrivateChannel & TextChannel.

Reply:
```suggestion
    * @arg {Array<String>} [content.stickerIDs] An array of IDs corresponding to stickers to send
```